local-env/setup-kind.sh

```diff
@@ -88,8 +90,40 @@ kubectl config set-context --current --user=oidc-user
 kubectl apply -k "${script_dir}/../config/crd/apiextensions.k8s.io/v1"
 kubectl apply -k "${script_dir}/../config/deployment"
 kubectl apply -k "${script_dir}/../config/user-rbac"
+kubectl apply -k "${script_dir}/../config/webhook"
+kubectl -n control-api delete svc webhook-service
+kind_network_gateway_ip="$(docker network inspect kind | jq -r '.[0].IPAM.Config[0].Gateway')"
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: webhook-service
+  name: webhook-service
+  namespace: control-api
+spec:
+  externalName: ${kind_network_gateway_ip}
+  selector:
+    app: webhook-service
+  type: ExternalName
+EOF
```

Comment:
This assumes that the control API runs on the host. However currently we deploy the control API to the created kind cluster. I don't think we want to change that do we?

Reply:
I didn't fully realize that. I'll update the PR.